### ISSUE TITLE
video library

### ISSUE THREAD

**Issue by devynirgang1** [processing/processing-video#209]

**Comment by benfry** [comment]
Looks like the video library may require a newer release than the beta you're using.

**Comment by Asmatzaile** [comment]
Yes, the first available release of the library for Processing 4 is compatible only with Processing 4.0b6 and newer.
See https://github.com/processing/processing-video/releases/tag/9

(@codeanticode, should this issue be closed?)
